src/app/components/task/ConditionalField.js

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import Select from '@material-ui/core/Select';
-import Typography from '@material-ui/core/Typography';
 import Input from '@material-ui/core/Input';
 import MenuItem from '@material-ui/core/MenuItem';
 import styled from 'styled-components';
 import { units } from '../../styles/js/shared';
+import CdsSelect from '../cds/inputs/Select';
```

Comment:
I have used this label for now because we need an alternative design approach to replace the material select still in this file, which is a multi-select with a pill design output